src/common/interflop.h

```diff
@@ -21,6 +31,37 @@ enum FCMP_PREDICATE {
   FCMP_TRUE,
 };
 
+typedef struct interflop_function_info
+{
+  // Indicate if the function is from library
+  char id[500];
+  // Indicate if the function is from library
+  char isLibraryFunction;
+  // Indicate if the function is intrinsic
+  char isIntrinsicFunction;
+  // Indicate if the function use float
+  char haveFloat;
+  // Indicate if the function use float
+  char haveDouble;
+  // mantissa precision for 64 bit float
+  int binary64_precision;
+  // mantissa precision for 64 bit float
+  int binary64_range;
+  // mantissa precision for 32 bit float
+  int binary32_precision;
+  // exponent precision for 32 bit float
+  int binary32_range;
+  // number of call for this function
+  int n_calls;
+}interflop_function_info_t;
+
+typedef struct interflop_function_stack
+{
+  interflop_function_info_t** functions;
+  int size;
+  int index;  
```

Comment:
What is the use of index here ?

Reply:
The new call stack implementation don't use index anymore